spec/helpers/session_timeout_warning_helper_spec.rb

```diff
@@ -20,29 +20,41 @@ def time_between_warning_and_timeout
   end
 
   describe '#timeout_refresh_url' do
-    before { expect(helper).to receive(:request).and_return(double(original_url: original_url)) }
+    before do
+      allow(helper).to receive(:request).and_return(
+        double(original_url: original_url, query_parameters: query_parameters)
+      )
+    end
+
+    let(:query_parameters) { { issuer: 'http://localhost:3000' } }
 
     context 'with no query in the request url' do
       let(:original_url) { 'http://test.host/foo/bar' }
 
-      it 'adds timeout=true params' do
-        expect(helper.timeout_refresh_url).to eq('http://test.host/foo/bar?timeout=true')
+      it 'adds timeout=true and sp_name params' do
```

Comment:
the description for this is a lie, it looks like `issuer` is added to the url and not `sp_name`